src/views/trips/RequestsView.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import CommonTable from './CommonTable';
+import Action from '../../components/Action';
+import Status from '../../components/Status';
+
+const tableHeads = [
+  { className: 'tripIdNav', attribute: 'Trip Id', key: 'tripId' },
+  { className: 'tripTypeNav', attribute: 'Trip Type', key: 'tripType' },
+  { className: 'createdAtNav', attribute: 'Created At', key: 'createdAt' },
+  { className: 'status', attribute: 'Status', key: 'status' },
+  { className: 'actionNav', attribute: 'Action', key: 'action' },
+];
+const navs = [
+  { className: 'tableHeader', attribute: <h2>My Travel Requests</h2>, key: 'tableHeader' },
+  { className: 'search', attribute: <input type="text" placeholder="search by date" />, key: 'search' },
+];
```

Comment:
I think both `tableHeads` and `navs` should be placed in the constants and then you import them.